src/database/entities/User.ts

```diff
@@ -0,0 +1,15 @@
+import { Cascade, Collection, Entity, OneToMany, PrimaryKey, Property } from "@mikro-orm/core";
+import { BaseEntity } from "./BaseEntity";
+import { Reminder } from "./Reminder";
+
+@Entity()
+export class User extends BaseEntity {
+  @PrimaryKey()
+  id: string;
+
+  @Property()
+  username?: string;
```

Comment:
are you using this property? Might be worth nothing that username will be unreliable to reference as they can be changed

Reply:
current stores the `username#4444` of the person who just requested. ultimately the userId is what's important. I had future plans for this property though. such as a last known name, or maybe a name history for the user.